### ISSUE TITLE
Integrate the Conway era

### ISSUE THREAD

**Issue by Jasagredo** [input-output-hk/ouroboros-network#3962]

**Comment by nfrisby** [comment]
Ledger Team has said we should start integrating Conway now. Because it's urgent, I'm going to try to bang it out myself, instead of leveraging it as an onboarding opportunity (as we did with Pawel for Alonzo).

Some notes:

- I approved PR #3933, with some minor caveats. The upstream Conway branch will depend on that.
- That upstream branch is currently labeled release/1.1.0 but Jared may be renaming it soon to release/1.1.x and then each concrete release 1.1.0, 1.1.1, etc would be a git tag.

This should be the simplest era integration yet. Therefore, once we've unblocked node, I might suggest each of the team members try to re-create the integration themselves, without looking at my code, as an onboarding exercise so people are exposed to what "adding an era" looks like.
